custom_components/ha_hatch/light.py

```diff
@@ -61,17 +66,27 @@ def turn_on(self, **kwargs):
             # If 100 is sent to Abode, response is 99 causing an error
             brightness = round(kwargs[ATTR_BRIGHTNESS] * 100 / 255.0)
         else:
-            brightness = self._attr_brightness
+            if isinstance(self.rest_device, RestIot):
+                brightness = 0
```

Comment:
why does this default to turning on at 0 brightness instead of the old known brightness when brightness is not specified?

Reply:
I can probably make this more clear with the changes you mentioned below, having the previous brightness here will allow us to pass that.